packages/cf.js/src/legacy/network.ts

```diff
@@ -1,7 +1,7 @@
 import ethers from "ethers";
 import * as _ from "lodash";
 
-import { Address } from "./utils";
+import { Address } from "./utils/index";
```

Comment:
Isn't `index` auto imported as default if you use `"./utils"`

Reply:
Yeah, blame my IDE. Fix pending